RInAppMessaging/Classes/Views/Components/FlexibleHeightImageView.swift

```diff
@@ -13,7 +13,7 @@ internal class FlexibleHeightImageView: UIImageView {
         let width = super.intrinsicContentSize.width
         let ratio = image.size.height / image.size.width
 
-        return CGSize(width: width, height: bounds.width * ratio)
+        return CGSize(width: width, height: bounds.width * ratio).integral
```

Comment:
Is it testable? It seems like it should be.

Reply:
I agree if it can be tested. It reminds me the screen resolution issue that occurred on the Analytics SDK.

Reply:
Added a test but not really sure if I'm testing the right thing. Seems like this kind of thing needs UI tests to verify the fix.

Reply:
thanks - logic tests like you've written is what I had in mind